ddtrace/contrib/flask/patch.py

```diff
@@ -270,38 +273,49 @@ def patch():
     setattr(flask, "_datadog_patch", True)
 
     Pin().onto(flask.Flask)
-    try:
-        from ddtrace.appsec.iast._taint_tracking import OriginType
+    if _is_iast_enabled():
```

Comment:
Github diff is not helping me visualize what's happening here. Did we just add a `_is_iast_enabled()` check and the rest of the changes are just a cooresponding indent?

Reply:
Yes we did :)